tests/functional/src/main/resources/META-INF/mule-module.properties

```diff
@@ -20,6 +20,17 @@ artifact.export.classPackages=org.mule.functional,\
                               org.mule.functional.util.sftp,\
                               org.mule.tck.util,\
                               org.mule.functional.api.classloading.isolation,\
+                              org.eclipse.aether.resolution,\
```

Comment:
Yes, it is required, I've create MULE-10514 to encapsulate this instead of exposing the Aether stuff.